### PR TITLE
Fix ejecting tanks from canisters not behaving as expected

### LINKED ISSUE #40865: Ejecting canisters from canisters dose not behave as expected
This happens from what i can tell because the canisters don't seem to be calling replace_tank when you use the eject button on the ui.

[Reproduction]: # 
Noticed it when filling plasma tanks frome the canister in the engineering bay 
place canister in the tank
fill to max pressure 
click eject button 
tank ends up on the ground even though your hand is empty

[For Admins]: # (Oddities induced by var-edits and other admin tools are not necessarily bugs. Verify that your issues occur under regular circumstances before reporting them.)

### PR DESCRIPTION
**Pull request by chris-hill1990**
:cl: Doomloard
fix: Fixed the canisters eject button to use the replace tank logic
fix: portable pumps eject button to use the replace tank logic
fix: portable scrubber eject button to use the replace tank logic
/:cl:

Fixes #40865 - when ever you eject the tanks from the canisters it would be placed on the ground not in the users hand

### CODE REVIEW

**Review by AnturK — DISMISSED** [review]

**Comment by AnturK** [comment]
You might want to remove "optional name here" from changelog.

**Comment by chris-hill1990** [comment]
Thanks for the heads up and cool my first pr. I sorry I had to fix it it bugged the hell out of me that it didn't behave as expected. Please let me know if I did everything right for the pr's and that

**Comment by chris-hill1990** [comment]
Just noticed this also happens in the portable air pumps

**Comment by AnturK** [comment]
Don't worry about travis failure it's issue with byond version.

File: code/modules/atmospherics/machinery/portable/canister.dm [hunk 1]
```diff
@@ -482,7 +482,6 @@
 				if(valve_open)
 					message_admins("[ADMIN_LOOKUPFLW(usr)] removed [holding] from [src] with valve still open at [ADMIN_VERBOSEJMP(src)] releasing contents into the <span class='boldannounce'>air</span><br>.")
 					investigate_log("[key_name(usr)] removed the [holding], leaving the valve open and transferring into the <span class='boldannounce'>air</span><br>", INVESTIGATE_ATMOS)
-				holding.forceMove(get_turf(src))
-				holding = null
+				replace_tank(usr, FALSE);
```
Comment (duncathan) [inline code/modules/atmospherics/machinery/portable/canister.dm]: please remove the semicolon

**Review by duncathan — CHANGES_REQUESTED** [review]
thank you very much for the fix! just get rid of that semicolon and you're good to go

**Pull request merged**